[2025-04-28T01-59-45-831Z] ERROR:
Name: Error
Message: UNHANDLED_REJECTION
Stack:
Error: UNHANDLED_REJECTION
    at Object.error (/home/container/events/logger.js:152:27)
    at process.<anonymous> (/home/container/index.js:210:10)
    at process.emit (node:events:518:28)
    at emitUnhandledRejection (node:internal/process/promises:252:13)
    at throwUnhandledRejectionsMode (node:internal/process/promises:388:19)
    at processPromiseRejections (node:internal/process/promises:475:17)
    at process.processTicksAndRejections (node:internal/process/task_queues:106:32)
Metadata: {
  "requestBody": {
    "files": [],
    "json": {
      "type": 4,
      "data": {
        "content": "Error al procesar la interacci√≥n",
        "tts": false,
        "enforce_nonce": false,
        "flags": 64
      }
    }
  },
  "rawError": {
    "message": "Unknown interaction",
    "code": 10062
  },
  "code": 10062,
  "status": 404,
  "method": "POST",
  "url": "https://discord.com/api/v10/interactions/1366232391672397904/aW50ZXJhY3Rpb246MTM2NjIzMjM5MTY3MjM5NzkwNDpxUHFLaVUxS0d4cG1aTHNRbXQwUm9aSm9vcG5nVmlyWG9KTlFvMEdkV3U4VnNVR1poajJFOXd5YWFwV3BzMFJqVDZ2Vk0xMGZLUmVNWHRybHhNMlpDZmZ1NVdGbTJsZzRNc0Fpc3dYUnVLcVJPYjRSN1hSbFNEOVRkVEJ5WUVaVA/callback?with_response=false"
}

